HOWTO.md

```diff
@@ -170,11 +170,11 @@ ssh -A -l ec2-user 54.247.190.179 -L8888:emqx-nlb-william-bbbbbbb.elb.eu-west-1.
 
 ``` bash
 # EMQX
-ssh -l ubuntu emqx-0.int.william
+ssh emqx-0
```

Comment:
Curiosity for my own information: if the default user in bastion is `ec2-user` and in those machines it's `ubuntu`, how `ssh emqx-0` works?

I tried here and indeed it worked, but I expected it to fail with something like `public key denied` error given the username difference. :thinking: